tests/container/test_heap.py

```diff
@@ -0,0 +1,124 @@
+from __future__ import absolute_import
+import random
+import sys
+import pytest
+import six
+from tchannel.container import heap
+
+from tchannel.container.heap import HeapOperation
+
+
+class IntHeap(HeapOperation):
+    def __init__(self):
+        self.values = []
+
+    def size(self):
+        return len(self.values)
+
+    def lt(self, i, j):
+        return self.values[i] < self.values[j]
+
+    def push(self, x):
+        self.values.append(x)
+
+    def pop(self):
+        n = len(self.values)
+        item = self.values[n - 1]
+        del self.values[n - 1]
+        return item
+
+    def swap(self, i, j):
+        self.values[i], self.values[j] = self.values[j], self.values[i]
+
+
+n = 1000
+
+
+@pytest.fixture
+def int_heap():
+    return IntHeap()
+
+
+def create_values(n):
+    values = []
+    for i in six.moves.range(n):
+        values.append(i)
+
+    random.shuffle(values)
+    return values
+
+
+def verify(ph, parent):
+    child1 = 2 * parent + 1
```

Comment:
Add a comment or docstring that says,

> Asserts that the heap property still holds.

Tests don't have to be devoid of docs. :)